### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by lalka13371488** [wg-easy/wg-easy#1534] [Q&A] [answered]
Hello, when creating 252 client config it gives an error (Internal Server Error). Apparently the limit for connection creation has been reached. Please tell me how to remove this limit or increase it. Thanks in advance.
![error](https://github.com/user-attachments/assets/c51f7d8c-c835-4290-b010-45ca30d4e313)

**Comment by kaaax0815** [comment]
Because of the limit of 253 clients with cidr /24.
will be fixed with 15.0.0 and #1333

**Reply by lalka13371488** [reply, marked as answer]
Thank you for your answer. Can you please tell me when the new release will be out?
